kpi/011-codelists-validation.adoc

```diff
@@ -2,55 +2,46 @@
 
 === Measurement
 
-Presence of valid terms from the referred codelist element.
+Each code value in the metadata is an exact match to the code in one of the authoritative codelists below.
 
-=== Rationale for measurement
+* ISO Codelists : https://standards.iso.org/iso/19139/resources/gmxCodelists.xml
```

Comment:
`ISO Codelists: https://standards.iso.org/iso/19139/resources/gmxCodelists.xml` (i.e. remove extra space between `ISO Codelists` and `:`.